src/game.cpp

```diff
@@ -8395,6 +8395,10 @@ void game::print_terrain_info( const tripoint &lp, WINDOW *w_look, int column, i
     } else {
         mvwprintw(w_look, line, column, _("%s; Movement cost %d"), tile.c_str(),
                   m.move_cost(lp) * 50);
+        float rawll = std::max(1.0, LIGHT_AMBIENT_LIT - g->m.ambient_light_at(lp) + 1.0);
```

Comment:
The `g->` is not needed. Without it, `this->m` will be used,  which is the same.